src/screens/TopicPageScreen/index.js

```diff
@@ -41,6 +41,7 @@ const TopicPageScreen = (props) => {
   const [userId, setUserId] = React.useState('');
   const [topicId, setTopicId] = React.useState('');
   const [isFollow, setIsFollow] = React.useState(params.isFollowing);
+  const [followType, setFollowType] = React.useState('');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A new state variable `followType` has been introduced. Make sure to handle its initial state and updates correctly throughout the component.
```diff
+ const [followType, setFollowType] = React.useState('');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->